website/docs/patterns/filter-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,220 @@
+
+![Filter dropdown with count](/assets/patterns/filter-patterns/filter-bar-dropdown-count.png =661x*)
+
+This, when combined with the applied filters provides detailed snapshot of what specific filter values are applied to the data set and the relevant parameters that they correspond with.
+
+**Communicating that multiple filter parameters have been applied**
+
+![Filter dropdown with multiple counts](/assets/patterns/filter-patterns/filter-bar-dropdown-count-multiple.png =683x*)
+
+## Overflow
+
+Depending on the complexity of the data set, it may be necessary to account for the overflow of elements within the filter bar or in the applied tags.
+
+### Filter bar overflow
+
+If the number of filterable parameters exceeds the available space, consider prioritizing the most commonly used filters in the filter bar and moving less important or more complex filters to a [Flyout](/components/flyout) that can be triggered via an action in the filter bar.
+
+**Hiding additional filters behind an action; in this scenario a secondary button**
+
+![More filters trigger](/assets/patterns/filter-patterns/overflow-filter-bar.png =559x*)
+
+**Triggering a Flyout with additional filter parameters**
+
+![Flyout triggered by more filters](/assets/patterns/filter-patterns/overflow-filter-bar-flyout.png =559x*)
+
+!!! Dont
+
+Don't stack multiple filter bars on top of a data set. This can result in unnecessary complexity in the UI.
+
+![Multiple filter bar rows](/assets/patterns/filter-patterns/overflow-filter-bar-multiple-rows.png =559x*)
+
+!!!
+
+### Filter sidebar overflow
+
+Similar to the guidelines for the filter bar, an action to view more filters within a [Flyout](/components/flyout) can be triggered within the sidebar orientation.
+
+**Hiding additional filters behind an action in a sidebar orientation**
+
+![More filters trigger in a sidebar](/assets/patterns/filter-patterns/overflow-sidebar-more-filters.png =559x*)
+
+!!! Info
+
+Consider each one of these scenarios carefully as moving additional filters to an interstitial component like a Flyout adds extra steps in the filtering process that might not be necessary.
+
+!!!
+
+### Applied filters overflow
+
+If the number of filterable parameters is large, there’s a likelihood that the number of applied filters may also extend to multiple lines. In this scenario the same methods apply as the filter bar and sidebar.
+
+Depending on the context of the filtering pattern, it may be helpful to display the applied filters to the user at all times. This can assist in easy dismissal of specific filters and help the user better parse how a data set is being filtered.
+
+**Extreme example of many applied filters**
+
+![Many filters applied](/assets/patterns/filter-patterns/overflow-applied-filters.png =559x*)
+
+**Applied filters within a Flyout**
+
+![Applied filters within a Flyout](/assets/patterns/filter-patterns/overflow-applied-filters-flyout.png =559x*)
```

Comment:
This interaction could be simplified. Have you considered using a "show more" action that reveals the rest of the filters in place instead of opening a flyout? 

This still feels a little like a "choose your own adventure," and we should also be opinionated here. 
Let's pick one and recommend that approach to work towards consistency across the UIs:
- Applied filters are always all visible.
- Only show a certain number of applied filters and then an action to display them in a Flyout (I do not recommend this approach; it feels unnecessarily complex and uncommon).
- Only show a certain number of applied filters and then reveal the rest in place via a "show more" action.